test/test-framework/deploy/crds/cache.example.com_dummys_crd.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
It is not part of this CR. Could you please remove these files?

Reply:
ok